.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: perl
+perl:
+    - "5.24"
```

Comment:
according to <https://travis-ci.org/binary-com/perl-Mojo-WebSocketProxy/jobs/135872261>, travis does not know about perl 5.24. This test is running with the system perl 5.14.